docs/v2/api/filers-api.md

```diff
@@ -0,0 +1,53 @@
+## Filers API
+
+### Filters
+
+* `/filers/<filing year>`
+
+   * `GET` - Returns a list of filers for the filing year defined in the URI.
```

Comment:
Should we mention that this currently only supports 2018 and you have to use a different url for 2017?